codec-http2/src/main/java/io/netty/handler/codec/http2/CompressorHttp2ConnectionEncoder.java

```diff
@@ -74,7 +74,7 @@
      */
     public CompressorHttp2ConnectionEncoder(Http2ConnectionEncoder delegate) {
         this(delegate, StandardCompressionOptions.brotli(), StandardCompressionOptions.gzip(),
-                StandardCompressionOptions.deflate());
+                StandardCompressionOptions.deflate(), StandardCompressionOptions.zstd());
```

Comment:
@normanmaurer I forgot to add `ZstdOptions` here. I am very sorry about it. The `zstd` lib has added the judgment of the compressed block. When the `zstd` library is updated, it can support the decompression of the block transmission data